user_management/api/tests/test_views.py

```diff
@@ -140,6 +140,13 @@ def test_delete_spacious_token(self):
         response = self.view_class.as_view()(request)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_unknown_token(self):
+        """An unknown token is accepted silently."""
+        auth = 'Token unknown'
+        request = self.create_request('delete', HTTP_AUTHORIZATION=auth)
+        response = self.view_class.as_view()(request)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
```

Comment:
The `GetAuthToken` view has throttles already: `UsernameLoginRateThrottle` and `LoginRateThrottle`.